src/Library/demos/Popovers/main.blp

```diff
@@ -0,0 +1,91 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Popovers";
+  description: _("Display content in a container above all other content on the window");
```

Comment:
```suggestion
  description: _("Display content in a container anchored to an other widget");
```

Reply:
Display content in a container anchored to another widget, right?

Reply:
Display content in a container anchored to another widget, right?

Reply:
Yes – did I make a mistake in my suggestion? I can't spot it.